lib/govuk_publishing_components/presenters/breadcrumb_selector.rb

```diff
@@ -55,6 +55,11 @@ def options(navigation)
             step_by_step: false,
             breadcrumbs: navigation.breadcrumbs,
           }
+        elsif navigation.content_is_a_corporate_information_page? && navigation.content_has_related_organisations?
```

Comment:
It might be more readable if you use a similar approach to on line 63 where we have
 `elsif navigation use_taxon_breadcrumbs?` ie a single method `use_organisation_breadcrumbs?`

Reply:
Fixed.